tests/unittests/evaluation_tests.py

```
@@ -24,8 +25,8 @@
 import fiftyone.utils.eval.segmentation as fous
 import fiftyone.utils.labels as foul
 import fiftyone.utils.iou as foui
-
-from decorators import drop_datasets
+import fiftyone.core.collections as foc
```

Ruff (0.8.2)

`fiftyone.core.collections` imported but unused

Remove unused import: `fiftyone.core.collections`

(F401)